scalameta/trees/shared/src/main/scala/scala/meta/Trees.scala

```diff
@@ -114,8 +114,8 @@ object Term {
     })
   }
   @ast class Block(stats: List[Stat]) extends Term {
-    // extension group block can have declarations without body too
-    checkFields(stats.forall(st => st.isBlockStat || st.is[Decl]))
+    // extension group block can have declarations or export without body too
+    checkFields(stats.forall(st => st.isBlockStat || st.is[Decl] || st.is[Export]))
```

Comment:
if this is allowed **only** in extensions, then i think it's time to change this to a `checkParent` call instead. adding something like this to `ParentChecks.scala`:
```
def TermBlock(tree: Term.Block, parent: Tree, dest: String): Boolean = {
  val isExportOK = parent.is[Defn.ExtensionGroup]
  tree.stats.forall {
    case _: Decl => true
    case _: Export => isExportOK
    case x => x.isBlockStat
  }
}
```
and then here
```
checkParent(ParentChecks.TermBlock)
```

Reply:
@kitbellew Thx, applied your suggestion!

Reply:
It seems that has broken the code :thinking: I think you might need to inline isExportOK

Reply:
` scala.meta.tests.parsers.dotty.MinorDottySuite.type-in-block` is failing

Reply:
fixed

Reply:
> if this is allowed **only** in extensions, then i think it's time to change this to a `checkParent` call instead. adding something like this to `ParentChecks.scala`:
> 
> ```
> def TermBlock(tree: Term.Block, parent: Tree, dest: String): Boolean = {
>   val isExportOK = parent.is[Defn.ExtensionGroup]
>   tree.stats.forall {
>     case _: Decl  | _: Export => isExportOK
>     case x => x.isBlockStat
>   }
> }
> ```
> 
> and then here
> 
> ```
> checkParent(ParentChecks.TermBlock)
> ```

looks like i relied on the comment too much (the one that @kpbochenek wrote a white ago, before @tgodzik added support for types anywhere). the comment needs to be changed to something like
```
// dotty block can have type declarations, or exports if within extension group, without body too
```

would this work:
```
def TermBlock(tree: Term.Block, parent: Tree, dest: String): Boolean = {
  val isExportOK = parent.is[Defn.ExtensionGroup]
  tree.stats.forall {
    case _: Decl  => true
    case _: Export => isExportOK
    case x => x.isBlockStat
  }
}
```